plugins/protocolbuffers/cpp/v29.2/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# syntax=docker/dockerfile:1.12
+FROM debian:bookworm-20241202 AS build
+
+ENV USE_BAZEL_VERSION=7.4.1
```

Comment:
Had to revert to previous Bazel version.

Reply:
Iirc we rarely had bazel issues with this version, maybe we should just stick to it and be less aggressive about bazel updates?